src/main.rs

```diff
@@ -142,8 +142,11 @@ fn main() {
             if args.store_genesis {
                 let near_config = indexer.near_config().clone();
                 actix::spawn(db_adapters::accounts::store_accounts_from_genesis(
-                    near_config,
+                    near_config.clone(),
```

Comment:
Can we use a reference to the config? (Don't spend more than 5 minutes on this task, keep the clone if the reference does not work well in this case)

Reply:
borrowed values doesn't live long enough and other stuff about borrowing for `'static`